README.MD

```diff
@@ -1,27 +1,41 @@
-# Cloud OCR SDK
+# Cloud OCR SDK 
 
 This .NET library is intended to simplify the development of image recognition solutions with ABBYY Cloud OCR SDK API. This repo contains the client library and the sample enabling simple access to high-quality recognition technologies.
+Using this library, you can supplement your C# code with ready-to-use solutions of images recognition with following export to the most popular text formats, such as searchable PDF, TXT, DOCX.
+
+
 For more information about the product visit [ABBYY Cloud OCR SDK website](https://www.ocrsdk.com/).
 
-## Quick start guide
+## Features
 
-To use the library in your project, do the following:
+- Text recognition
+  - full-page and zonal OCR (printed text) for 200+ languages
+  - ICR (hand-printed text)
+- Document conversion
+  - convert image/PDF to searchable PDF, PDF/A and Microsoft Word, Excel, PowerPoint
+- Data extraction
+  - Receipt recognition
+  - Barcode recognition 
+  - Business card recognition
+  - ID recognition (MRZ)
+- .Net Framework и .Net Core support
+  - integrate this library into .Net, ASP.NET or other applications
 
-1. [Register](https://cloud.ocrsdk.com/Account/Register) on the ABBYY Cloud OCR SDK website and create your Application. You will provide the Application ID and Application Password as authentication information to initialize the top hierarchy object of the library and to get access to the API.
-2. Install the package from the [NuGet.org](https://www.nuget.org/packages/Abbyy.CloudSdk.V2.Client/) and it will update automatically. If you intend to modify the API, you can download the library from this repo.
-3. Implement your application using ABBYY OCR and capturing technologies.
 
-## Documentation
 
-Find the full API description in the [Documentation](https://www.ocrsdk.com/documentation/api-reference/) section.
+## Quick start guide
 
-## Sample
+To use the library in your project for image processing, do the following:
 
-The sample demonstrates how to process image with the specified parameters and export the result to TXT and DOCX using the ABBYY Cloud OCR SDK client library. Start your developement with the following steps.
+1. [Register](https://cloud.ocrsdk.com/Account/Register) on the ABBYY Cloud OCR SDK website and create your Application. You will provide the Application ID and Application Password as authentication information to initialize the top hierarchy object of the library and to get access to the API.
+2. Download the library from this repo, if you intend to modify it, or simply install the package from the [NuGet.org](https://www.nuget.org/packages/Abbyy.CloudSdk.V2.Client/).
+3. Implement your application using ABBYY OCR and capturing technologies.
 
-Provide your authentication info and the HTTP server URL address, used for API calls (see [Data processing location](https://www.ocrsdk.com/documentation/specifications/data-processing-location/) for details):
```

Comment:
Why we remove Data processing location link?

Reply:
Actually, we do not.